### ISSUE TITLE
Moving chairs should be a non-issue

### ISSUE THREAD

**Issue by fantasai** [w3c/TPAC2022-Health-rules#5]
> In each meeting room: the attendees will remain seated according to the room setup. Chairs and tables will not be moved around.

Social distancing is not particularly helpful when everyone is masked and in an indoor setting for hours at a time, and enforcing it like this is going to be more annoying than helpful. Let participants put their chairs and tables wherever they think is best.

**Comment by alexandralacourba** [comment]
We needed to plan this rule, in order to maintain social distancing. Although people will have masks, and although food will not allowed in meeting rooms, we should also plan that people may need to remove their masks for a short period of time if they need to drink.
Although, there is another reason, which is not health related: we will set up the rooms to make sure that remote people are included to the discussion. Therefore the set up should not be modified by groups.

**Comment by fantasai** [comment]
@alexandralacourba It's perfectly normal and should be acceptable for attendees to move chairs around during breaks as needed to have discussions. If there are particular concerns about remote attendance, you can explain the requirements to the chairs and they can manage the room. The event shouldn't be making rules that aren't important to follow, it decreases the attention spent on the rules that matter.

**Comment by alexandralacourba** [comment]
@fantasai, I totally understand your point, but this is something we need to think about especially if people end us drinking coffee in the meeting room. We need to finalize this point.

**Comment by fantasai** [comment]
@alexandralacourba If masking is _required_, then we can't be drinking coffee and chatting in the meeting room, it kinda defeats the point. :)

**Comment by alexandralacourba** [comment]
Yes, I agree but in practical it is challenging to ask people to not drink at all during several hours in a row. Breaks should not be taken in the rooms for sure. We need to set up a clear plan outside the breaks time so that it is acceptable to everybody.

**Comment by fantasai** [comment]
@alexandralacourba Yes, I understand. I just don't want to turn the meeting spaces into break spaces. :) Proposed plan:

- Don't provide water at each seat in the rooms: if someone needs it, they can bring something to their seat to drink, but make it an intentional choice to bring drinks into the room rather than a casual default.
- MAY remove mask while actively drinking or eating in a meeting room, but
- MUST wear a mask at all times while talking indoors, unless making accommodations for disabilities

COVID emission is roughly 10x or more while speaking vs breathing silently, so I think if we make that last point really clear, it'll help minimize the danger.

**Comment by alexandralacourba** [comment]
@fantasai, I'm sorry for my late reply as I was on vacation. Yes, we do not intend to provide water at each seat. People will be able to remove masks to drink, if necessary. We do not intend to permit eating in meeting rooms, as a general rule

**Comment by frivoal** [comment]
The [TPAC health rules page](https://www.w3.org/2022/09/TPAC/health.html) is *very* long. The longer it gets, the higher the chance that someone will misread them. Or skim/skip altogether. Or fail to remember. (Or that the rules will contradict themselves, see #13.)

I think they would greatly benefit from a deliberate effort at making them shorter. Removing things that don't matter much, such as the sentence that @fantasai pointed out about chairs, would help.

**Comment by alexandralacourba** [comment]
We should keep it, as it is important for several aspects:
- Social distancing
- AV set for remote participation
- AV setup for security reasons

all the reasons are not related to health, I reckon, but this is important that all attendees are aware of the aspect
